File: EmojiPanel/src/main/java/com/binlee/emoji/DebugApplication.java
package com.binlee.emoji;

import android.annotation.SuppressLint;
import android.app.Application;
import android.content.Context;

public class DebugApplication extends Application {

    @SuppressWarnings("Reflections")
    @SuppressLint("StaticFieldLeak")
    private static Application sApp;

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        sApp = this;
    }

    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: EmojiPanel/src/main/java/com/binlee/emoji/DebugApplication.kt
package com.binlee.emoji

import android.annotation.SuppressLint
import android.app.Application
import android.content.Context


class DebugApplication : Application() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        sApp = this
    }

    override fun onCreate() {
        super.onCreate()
    }

    companion object {
        @SuppressLint("StaticFieldLeak")
        private var sApp: Application? = null
    }
}